shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -0,0 +1,72 @@
+use cassandra_cpp::*;
+use std::thread;
+use std::thread::sleep;
+use std::time::Duration;
+use tracing::info;
+
+mod basic_driver_tests;
+
+pub struct CassandraTestContext {
```

Comment:
The `CassandraTestContext` wrapper doesnt add anything over just a raw `Session`.
Lets remove it and return the `Session` directly.

Reply:
Doing so would reduce the size of this PR.